stack.yaml

```diff
@@ -3,3 +3,4 @@ resolver: nightly-2017-01-25
 packages:
   - "."
   - "./docs/source/tutorial"
+  - "./graphql-wai"
```

Comment:
Hmm.  I guess we should move to a mono-repo structure officially. Not in this PR.